packages/frontend/src/components/Election/Results/DowloadCSV.tsx

```diff
@@ -0,0 +1,62 @@
+import { StyledButton } from "~/components/styles";
+import { CSVLink } from "react-csv";
+import { useState } from "react";
+import { Election } from "@equal-vote/star-vote-shared/domain_model/Election";
+import { Ballot } from "@equal-vote/star-vote-shared/domain_model/Ballot";
+
+interface Props {
+	data: {
+		election: Election;
+		ballots: Ballot[];
+	};
+}
+export const DownloadCSV = ({data}: Props) => {
+	const [csvData, setcsvData] = useState([])
+    const [csvHeaders, setcsvHeaders] = useState([])
+	const buildCsvData = () => {
+        let header = [
+            { label: 'ballot_id', key: 'ballot_id' },
+            ...data.election.races.map(race => race.candidates.map(c => ({ label: `${race.title}!!${c.candidate_name}`, key: `${race.race_id}-${c.candidate_id}` })))
+        ]
+		header = header.flat()
+        let tempCsvData = data.ballots.map(ballot => {
+            let row = {ballot_id: ballot.ballot_id}
+            ballot.votes.forEach(vote => vote.scores.forEach(score => {
+                row[`${vote.race_id}-${score.candidate_id}`] = score.score
+            }));
+            return row
+        })
+        setcsvHeaders(header)
+        setcsvData(tempCsvData)
+        return false
```

Comment:
Does this line do anything?

Reply:
Looks like it was copied over from my old code. It looks like that old function probably used to return a valid indicator before continuing to something else. Probably fine to remove it.